omero/conf.py

```diff
@@ -176,4 +176,5 @@
 ]
 
 exclude_patterns = ['sysadmins/unix/walkthrough/requirements*',
-                    'downloads/inplace', 'downloads/cli']
+                    'downloads/inplace', 'downloads/cli',
+                    'changelog.rst']
```

Comment:
Why the exclusion? What's expected to render it?

Reply:
That file is not used anywhere it is only used so it can be "injected" in the history